### PR TITLE
Add Rusty's Real Deal Baseball

### PR DESCRIPTION
**Pull request by solitonmedic**
Adds Rusty's Real Deal Baseball.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/rustys-real-deal-baseball/game.dat [hunk 1]
```diff
@@ -0,0 +1,20 @@
+title = "Rusty's Real Deal Baseball"
+description = "Rusty's Real Deal Baseball is a baseball video game developed and published by Nintendo, starring the titular Rusty Slugger and the player's Mii character. It features a collection of baseball themed minigames. Minigames are purchased and downloaded from the Nintendo eShop. Players can haggle with Rusty to decrease a game's price with donuts or discount tickets obtained from the minigames. Certain minigames advance the game's story."
```
Comment (CodingKoopa) [inline games/rustys-real-deal-baseball/game.dat]: "It features a collection of baseball themed minigames. Minigames are purchased and downloaded from the Nintendo eShop." -> "It features a collection of baseball themed minigames that are purchased and downloaded from the Nintendo eShop.

Comment (CodingKoopa) [inline games/rustys-real-deal-baseball/game.dat]: This is a bit uneededly long, I'd say you can take the last sentence out.

**Comment by CodingKoopa** [comment]
I removed the apostrophe from your wiki title. The wiki title correlates with the internal folder name of `rustys-real-deal-baseball`, and not the value for the `title` key of *Rusty's Real Deal Baseball*.

File: games/rustys-real-deal-baseball/game.dat (hunk 1 above)
Comment (solitonmedic) [reply games/rustys-real-deal-baseball/game.dat]: It doesn't. You'd still need to connect to the internet and to the eShop to download the actual game files.

For now, all it does is provide a actual Mii character to replace the headless one.

Comment (CodingKoopa) [reply games/rustys-real-deal-baseball/game.dat]: Ah, ok, that's fine then.

**Pull request merged**